### ISSUE TITLE
The warnings for additional fields (even the _ language ones) appear on the top of the page at http://standard.open-contracting.org/validator/

### ISSUE THREAD

**Issue by daniel-clair** [OpenDataServices/cove#348]
Most of the people see the warnings as errors, then please be careful on the use of Warning.
In the case of "_language" fields shouldn't be Warnings, should be NOTICEs and if possible they shouldn't be ON TOP (like something is WRONG and it needs to be fixed)

Please review the decision of putting this on the TOP of the page (or at least don't include the _language fields on this list).

**Comment by robredpath** [comment]
Thanks for reporting this @daniel-clair - and sorry for not responding sooner. 

There's two things that we're planning in order to make this clearer: 
- We've changed the layout on the version of CoVE that validates 360Giving data; it's still experimental but should be going live in the next week or so. We've rearranged things so that it's much clearer if your data is valid or not, and any additional information (eg fields in schema but not in your data) come after the notice that your data is valid. We've got #401 to handle applying similar changes to the OCDS validator. 
- We're reviewing how we handle the _language fields in particular; they shouldn't generate warnings, but it'll take some coding for the validator to recognise them as being a special case. I've added #402 if you'd like to track progress on this.
